zerver/webhooks/jotform/fixtures/response.multipart

```diff
@@ -65,7 +65,7 @@ Content-Disposition: form-data; name="customBody"
 --------------------------J1JNT4vbVO4FCC1qLG4HSv
 Content-Disposition: form-data; name="rawRequest"
 
-{"slug":"submit\/243231271343446","jsExecutionTracker":"build-date-1732271172685=>init-started:1732615894703=>validator-called:1732615894706=>validator-mounted-false:1732615894706=>init-complete:1732615894707=>onsubmit-fired:1732615897940=>observerSubmitHandler_received-submit-event:1732615897941=>submit-validation-passed:1732615897942=>observerSubmitHandler_validation-passed-submitting-form:1732615897944","submitSource":"form","buildDate":"1732271172685","uploadServerUrl":"https:\/\/upload.jotform.com\/upload","eventObserver":"1","q3_studentsName":{"first":"Niloth","last":"P"},"q33_typeOf":"Online Tutoring","q34_subjectFor":"Math","q35_grade":"12","q28_appointment":{"implementation":"new","date":"","duration":"60","timezone":"Asia\/Calcutta (GMT+06:30)"},"timeToSubmit":"3","preview":"true","validatedNewRequiredFieldIDs":"{\"new\":1}","path":"\/submit\/243231271343446"}
+{"slug":"submit\/243231271343446","jsExecutionTracker":"build-date-1732271172685=>init-started:1732615894703=>validator-called:1732615894706=>validator-mounted-false:1732615894706=>init-complete:1732615894707=>onsubmit-fired:1732615897940=>observerSubmitHandler_received-submit-event:1732615897941=>submit-validation-passed:1732615897942=>observerSubmitHandler_validation-passed-submitting-form:1732615897944","submitSource":"form","buildDate":"1732271172685","uploadServerUrl":"https:\/\/upload.jotform.com\/upload","eventObserver":"1","q3_studentsName":{"first":"Niloth","last":"P"},"q33_typeOf":"Online Tutoring","q34_subjectFor":"Math","q35_grade":"12","q21_priorTutoring":"No","q28_appointment":{"implementation":"new","date":"","duration":"60","timezone":"Asia\/Calcutta (GMT+06:30)"},"timeToSubmit":"3","temp_upload":{"q11_studentId":["ID.png#jotformfs-e4f4ece4d0a90#0193ca5e-ba20-79cb-be4a-afd39a0fe696"],"q13_reportCard":["Report Card Niloth.pdf#jotformfs-e4f4ece4d0a90#0193ca5e-cba0-7409-b60e-494c455564c3"]},"file_server":"jotformfs-e4f4ece4d0a90#0193ca5e-cba0-7409-b60e-494c455564c3","preview":"true","validatedNewRequiredFieldIDs":"{\"new\":1}","path":"\/submit\/243231271343446","studentId":["https:\/\/www.jotform.com\/uploads\/UserNiloth\/243231271343446\/6100762336995170500\/ID.png"],"reportCard":["https:\/\/www.jotform.com\/uploads\/UserNiloth\/243231271343446\/6100762336995170500\/Report%20Card%20Niloth.pdf"]}
```

Comment:
I think it would be nice to have more than one image and more than one non-image file. And maybe a field with multiple linked files.
I tested the above cases manually, but it would help to have automated tests ensure the same.
I like how you've tried out different file extensions, it would help to do the same to maintain variety.